portfolio/src/main/webapp/old/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```

Comment:
Do you need the files that are in the old/ directory? If not, you can delete them as you don't want to merge them to your main branch.